docs/data/date-pickers/adapters-locale/adapters-locale.md

```diff
@@ -115,7 +105,7 @@ function App({ children }) {
 
 :::
 
-## 12h/24h format
+## Meridiem — 12h/24h format
```

Comment:
Should improve Algolia search.
For now, searching "meridiem" do not lead to this section which is problematic.